tests/data/DISABLED

```diff
@@ -55,7 +55,10 @@
 1533
 1540
 1591
+# Test that are disabled here for Cirrus CI Windows builds
+%if Windows
```

Comment:
This disables the test on **all** windows builds, not just the Cirrus ones!

Reply:
Any idea to do this only on Cirrus ones?

Reply:
As discussed in #8920, it does seem to work on appveyor for example. I prefer we first try to actually fix the test case.